metrics/report/report_dockerfile/parallel.R

```diff
@@ -57,37 +56,35 @@ for (currentdir in resultdirs) {
 			# convert ms to seconds
 			cdata=data.frame(boot_time=as.numeric(fdata$BootResults$launch_time$Result)/1000)
 			cdata=cbind(cdata, delete_time=as.numeric(fdata$BootResults$delete_time$Result)/1000)
+			cdata=cbind(cdata, npod=as.numeric(fdata$BootResults$n_pods$Result))
 
 			# If we have more than 20 items to draw, then do not draw the points on
 			# the graphs, as they are then too noisy to read.
 			if (length(cdata[, "boot_time"]) > 20) {
 				skip_points=1
 			}
 
-			cdata=cbind(cdata, count=seq_len(length(cdata[, "boot_time"])))
 			cdata=cbind(cdata, testname=rep(testname, length(cdata[, "boot_time"]) ))
 			cdata=cbind(cdata, dataset=rep(datasetname, length(cdata[, "boot_time"]) ))
 
 			# Store away as a single set
 			data=rbind(data, cdata)
-
-			count = count + 1
 		}
 	}
 }
 
 # Show how boot time changed
 boot_line_plot <- ggplot() +
-	geom_line( data=data, aes(count, boot_time, colour=testname, group=dataset), alpha=0.2) +
-	geom_smooth( data=data, aes(count, boot_time, colour=testname, group=dataset), se=FALSE, method="loess", size=0.3) +
+	geom_line( data=data, aes(npod, boot_time, colour=testname, group=dataset), alpha=0.2) +
+	geom_smooth( data=data, aes(npod, boot_time, colour=testname, group=dataset), se=FALSE, method="loess", size=0.3) +
```

Comment:
putting the aes portion in ggplot would remove the need for duplication. If we remove the smoothing, it wouldn't matter. Not a blocking comment, just a note

Reply:
Ack. I'll try to remember to clean and dedup the aes's during further edits. I wish I could remember why I split them out in the first place... but it was a long time ago in a project far far away...

Reply:
you can pass different things to aes, maybe that was the reason?